conda_smithy/templates/README.md.tmpl

```diff
@@ -71,10 +71,22 @@ Current build status
 ====================
 {%- set appveyor_badge_name = github.repo_name.replace('_', '-') %}
 {%- set appveyor_url_name = appveyor_badge_name.replace('.', '-') %}
-
+{# #}
+{%- if "circle" in ci_enabled %}
 Linux: [![Circle CI](https://circleci.com/gh/{{github.user_or_org}}/{{github.repo_name}}.svg?style=svg)](https://circleci.com/gh/{{github.user_or_org}}/{{github.repo_name}})
+{%- else %}
+Linux: ![](https://cdn.rawgit.com/{{github.user_or_org}}/{{github.repo_name}}/master/ci_support/disabled.svg)
```

Comment:
> You are right, but honestly I'm not worried about it.

Sorry, I'm probably being overly concerned about this. I guess I just don't want to be dealing with issues because of this badge.

> We can always hold a copy of the SVG in the location for a period of time to give us grace on moving the feedstocks over.

Yeah, that is definitely the other option and I thought about that too. I guess I wanted to free us from thinking about such silly things. There are enough problems we have to handle at any one time that I would rather the badges not be one. 😉

> I don't think we need to bikeshed this one

Sorry about that. Somehow I knew this PR was going to result in bikeshedding. Didn't realize I was going to be the one doing it though.

> personally I'd rather be able to move quickly on updating the SVG than worry about pointing at a specific release.

Yeah, I see your point. Honestly I think I'm being overly worried about the badge changing or moving. It probably is going to sit there and be the same.

> Either way works for me though - you can have the final call on it.

Ok, I have picked the commit the badge was added in. That seems to work ok FWICT. Will show in an example.

Also, did some clean-up so that the URL can be set in one place.